invenio_stats/contrib/events.py

```diff
@@ -23,12 +23,25 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 """Registration of contrib events."""
+from invenio_search import current_search_client
+
 from invenio_stats.indexer import EventsIndexer
+from invenio_stats.tasks import StatAggregator
 
 
-def register():
+def register_events():
     """Register sample events."""
     return [dict(event_type='file-download',
                  processor=EventsIndexer),
             dict(event_type='record-view',
                  processor=EventsIndexer)]
+
+
+def register_aggregations():
```

Comment:
The file is name `events.py`. We should find a more appropriate name if we also put the aggregations there.